components/layout_2020/fragment_tree/box_fragment.rs

```diff
@@ -297,7 +289,7 @@ impl BoxFragment {
 
     pub(crate) fn calculate_resolved_insets_if_positioned(
```

Comment:
I think we can go a bit further here and have this return `AuOrAuto`. That will require some more changes such `resolved_sticky_insets` being `Option<PhysicalSides<AuOrAuto>>`, but it's good to convert the entire struct at once.